mmv1/products/filestore/Instance.yaml

```diff
@@ -336,3 +336,65 @@ properties:
       resource, see the `google_tags_tag_value` resource.
     immutable: true
     ignore_read: true
+  - name: 'initialReplication'
+    api_name: replication
+    type: NestedObject
+    description: |
+      Replication configuration, once set, this cannot be updated.
+    url_param_only: true
+    immutable: true
+    properties:
+      - name: 'role'
+        type: Enum
+        description: |
+          The replication role.
+        default_value: "STANDBY"
+        enum_values:
+          - 'ROLE_UNSPECIFIED'
+          - 'ACTIVE'
+          - 'STANDBY'
+      - name: 'replicas'
+        type: Array
+        description: |
+          The replication role.
```

Comment:
Could we clarify usage in the description here? i.e. "This should be specified on the replica, marking the active as the `peer_instance`" or similar.